dissectors/pro-dj-link-status.lua

```diff
@@ -121,13 +318,95 @@ function p_pdj_status.dissector (buf, pkt, root)
     subtree_flags:add(pdj_status_f.flags_onair, flags_ptr)
     subtree_flags:add(pdj_status_f.flags_bpm, flags_ptr)
 
+    local pitch1_ptr = buf(0x8c,4)
+    local pitch1 = pitch1_ptr:uint()
+    local pitch1_percent = ((pitch1_ptr:uint() - 0x100000) / 0x100000) * 100  -- for use with BPM, below
+    local pitch1_display = string.format("%+.2f", pitch1_percent)
+    subtree:add(pdj_status_f.t0a_pitch1, pitch1_ptr, pitch1, nil, "(" .. pitch1_display .. "%)")
+
+    local pitch2_ptr = buf(0x98,4)
+    local pitch2 = pitch2_ptr:uint()
+    local pitch2_display = string.format("%+.2f", ((pitch2 - 0x100000) / 0x100000) * 100)
+    subtree:add(pdj_status_f.t0a_pitch2, pitch2_ptr, pitch2, nil, "(" .. pitch2_display .. "%)")
+
+    local pitch3_ptr = buf(0xc0,4)
+    local pitch3 = pitch3_ptr:uint()
+    local pitch3_display = string.format("%+.2f", ((pitch3 - 0x100000) / 0x100000) * 100)
+    subtree:add(pdj_status_f.t0a_pitch3, pitch3_ptr, pitch3, nil, "(" .. pitch3_display .. "%)")
+
+    local pitch4_ptr = buf(0xc4,4)
+    local pitch4 = pitch4_ptr:uint()
+    local pitch4_display = string.format("%+.2f", ((pitch4 - 0x100000) / 0x100000) * 100)
+    subtree:add(pdj_status_f.t0a_pitch4, pitch4_ptr, pitch4, nil, "(" .. pitch4_display .. "%)")
+
+    local bpm_ptr = buf(0x92,2)
+    local bpm = bpm_ptr:uint()
+    local effective_tempo = bpm / 100 * (100 + pitch1_percent) / 100
+    local bpm_display = string.format("(%.1f bpm; effective tempo: %.1f bpm)", bpm / 100, effective_tempo)
+    subtree:add(pdj_status_f.t0a_bpm, bpm_ptr, bpm, nil, bpm_display)
```

Comment:
```suggestion
    local bpm_ptr = buf(0x92,2)
    local bpm = bpm_ptr:uint()
    local bpm_display = "(Unknown)"
    if bpm ~= 0xFFFF then
      bpm_display = bpm / 10
      effective_tempo = bpm / 100 * (100 + pitch1_percent) / 100
      bpm_display = string.format("(%.1f bpm; effective tempo: %.1f bpm)", bpm / 100, effective_tempo)
    end
```

Need to apply the same logic here to check for `0xffff` bpm.